foundations/html_css/html-foundations/links-and-images.md

```diff
@@ -139,7 +139,7 @@ Think of your domain name (`town.com`) as a town, the directory in which your we
 
 Websites would be fairly boring if they could only display text. Luckily HTML provides a wide variety of elements for displaying all sorts of different media. The most widely used of these is the image element.
 
-To display an image in HTML we use the `<img>` element. Unlike the other elements we have encountered, the `<img>` element is self-closing. Empty, self-closing HTML elements do not need a closing tag.
+To display an image in HTML we use the `<img>` element. Like the `<meta>` element, the `<img>` element is self-closing. Empty, self-closing HTML elements do not need a closing tag.
```

Comment:
Can we simplify this instead of referring back to the meta tag instead?

```suggestion
To display an image in HTML we use the `<img>` element. The `<img>` element is self-closing. Empty, self-closing HTML elements do not need a closing tag.
```